cmd/ncproxy/run.go

```diff
@@ -159,6 +165,29 @@ func run() error {
 		}
 	}
 
+	// setup ncproxy databases
+	if *dbPath == "" {
+		// default location for ncproxy database
+		binLocation, err := os.Executable()
+		if err != nil {
+			return err
+		}
+		*dbPath = filepath.Dir(binLocation)
```

Comment:
The comment for `database-path` sounds like it's a file, but this seems to be a directory. Should we just be taking a file path instead?